microsoftbotframework/response.py

```diff
@@ -133,7 +134,7 @@ def _set_header(self):
             self.headers = {"Authorization": "{} {}".format(token["token_type"], token["access_token"]),
                             "User-Agent": "Microsoft-BotFramework/3.1 (BotBuilder Node.js/3.7.0)"}
 
-    def _request(self, response_url, method, response_json=None):
+    def _request(self, response_url, method, response_json=None,  timeout_seconds=None):
         self._set_header()
```

Comment:
- in _request please pass timeout_seconds into _set_header
- in _set_header please pass timeout_seconds to _get_remote_auth_token
- in _set_header please pass timeout_seconds to _get_redis_auth_token
- in _get_redis_auth_token please pass timeout_seconds to _get_remote_auth_token

Reply:
Hi Justin I think no need to pass the timeout_seconds for every method just I changed the code passing only with Response class and it will do self. timeout_seconds = kwargs.pop('timeout_seconds', None) then it will get used anywhere in this class.
Please let me know if I'm wrong.